packages/web-twig/src/Resources/components/Tooltip/Tooltip.twig

```diff
@@ -13,27 +13,30 @@
 {%- set _rootPlacementClassName = _spiritClassPrefix ~ 'Tooltip--' ~ _placement -%}
 
 {# Attributes #}
-{%- set _idAttr = _id ? 'id=' ~ _id : null -%}
+{%- set _idAttr = _id ? 'id="' ~ _id | escape('html_attr') ~ '"' : null -%}
+{%- set _ariaControlsAttr = _id ? 'aria-controls="' ~ _id | escape('html_attr') ~ '"' : null -%}
+{%- set _dataTargetAttr = _id ? 'data-target="#' ~ _id | escape('html_attr') ~ '"' : null -%}
 
 {# Miscellaneous #}
 {%- set _styleProps = useStyleProps(props) -%}
 {%- set _classNames = [ _rootClassName, _rootPlacementClassName, _rootDismissibleClassName, _styleProps.className ] -%}
 {%- set _mainPropsWithoutId = props | filter((value, prop) => prop != 'id') -%}
+{%- set _allowedAttributes = [ 'id' ] -%}
 
 <div
-    {{ mainProps(_mainPropsWithoutId) }}
+    {{ mainProps(_mainPropsWithoutId, _allowedAttributes) }}
```

Comment:
what is the purpose of the `_allowedAttributes` here? We have id already in the props API?

Reply:
Removed.